src/query/expression/src/utils/display.rs

```diff
@@ -245,6 +252,10 @@ impl<'a> Display for ScalarRef<'a> {
                 let value = jsonb::to_string(s);
                 write!(f, "{value}")
             }
+            ScalarRef::Geometry(s) => {
+                let geom = Ewkb(s.to_vec()).to_ewkt(None).unwrap();
+                write!(f, "{geom:?}")
```

Comment:
Add quotes for Geometry to be presented as a string, because `ScalarRef` will be used to store the default value of fields, if the default value is not a string, panic will occur when parsing.